.github/workflows/ci-nightly.yml

```
@@ -28,19 +28,23 @@ jobs:
       run: |
         git config --global user.name "${{ env.GITHUB_SERVICE_USER }}"
         git config --global user.email "${{ env.GITHUB_SERVICE_EMAIL }}"
-    
+   
     - name: Install package dependencies / prepare workspaces
       run: yarn install --frozen-lockfile
+    
+    - name: check for change
+      run: yarn check
 
     - name: Testing Prettier format
       run: lerna run prettier:diff
 
     - name: Testing unit tests
       run: lerna run test --stream   
-      
+
+    - name: look for change file
```

This one is probably more semantic and helps to differentiate with "check for changed files" as we want to exit if there are *now* generated files.
```suggestion
    - name: check for changed files
```

---

Looks like I was mistaken on what's being executed here. This isn't running a beachball script, the script in question is actually checking git status...https://github.com/microsoft/fluentui/blob/master/scripts/tasks/check-for-modified-files.ts

---

I agree. I just remove it. I think we need to check the change in daily build though.

---

Yep - the PR instance should have a single check after yarn gets dependencies installed.